__tests__/app.test.js

```diff
@@ -102,4 +91,47 @@ describe('/api', () => {
     })
 })
 
+describe('/api/articles', () => {
+    test('200: GET /api/articles - responds with an array of article objects containing the relevant keys', () => {
+        return request(app)
+        .get('/api/articles')
+        .expect(200)
+        .then(({ body }) => {
+            const {articles} = body;
+            expect(articles.length > 0).toBe(true);
```

Comment:
this is fine but there's a few other jest matchers you can use here like .toHaveLength, .toBeGreaterThan, but I would personally just assert against the actual length itself here